caffe2/mpi/mpi_ops_gpu.cc

```diff
@@ -50,6 +66,15 @@ namespace caffe2 {
 #define CAFFE2_HAS_CUDA_MPI_ALLREDUCE 0
 #endif // CAFFE2_FORCE_FALLBACK_CUDA_MPI
 
+// We allow a macro to force using CUDA functions
+#ifdef CAFFE2_FORCE_CUDA_MPI
```

Comment:
s/CAFFE2_FORCE_CUDA_MPI/CAFFE2_USE_CUDA_MPI?

Reply:
This is a typo. That should be CAFFE2_USE_CUDA_MPI.